### ISSUE TITLE
Economic impacts % figures don't update on year selection

### ISSUE THREAD

**Issue by shawnbot** [18F/doi-extractives-data#1953]
## Expected Behavior
The % of a state's GDP from extractives under the chart on state pages doesn't update when you select a different year.

## Actual Behavior
The ", or X% of Y's GDP" should either update or be modified to remove the percentage.

## Steps to reproduce the behavior
[Delaware](https://federalist.18f.gov/preview/18F/doi-extractives-data/dev/explore/DE/#gdp) is a good example because it has no 2015 data, so when you select a year with data the percentage always reads as `--`:

![image](https://cloud.githubusercontent.com/assets/113896/18561179/a042f1e4-7b33-11e6-9cdc-6931d9ff04f7.png)

**Comment by coreycaitlin** [comment]
See also #1879

**Comment by coreycaitlin** [comment]
If it's not prohibitively hard, it would be great to get this fixed — I moved the percentage into this sentence because, for GDP and wage/salary jobs, it's an incredibly helpful indicator of whether I should care about the numbers. For a while, we tried only including the percentage for the most recent year's data, and moving that information to the intro section above, but that created other consistency/usability/information hierarchy issues, because it then didn't update with the drop-down year selector. I moved it back under the chart in my most recent review of the overall page structure, because it's more parallel with other sections.

We also include the % GDP (and sometimes jobs) in the overview/intro of the page, and I really hate putting content like that in an intro without also including it in-context, because it forces users to scroll waaaaaay up and down the page to connect the % number with the # number (or leaves them feeling vaguely like they saw it once but can't find it again).
